CHANGELOG.md

```diff
@@ -27,7 +27,7 @@
 - Added metrics for the vsock device.
 - Added devtool strip command which removes debug symbols from the release
   binaries.
-
+- Allow no space after ":"parsing header
```

Comment:
```suggestion
- Any number of whitespace characters are accepted after ":" when parsing HTTP
  headers.
```

Reply:
@lauralt Sorry... I added one more newline.

Reply:
Please update this line :D